pkg/cmd/run.go

```diff
@@ -134,9 +135,13 @@ func RunMicroshift(cfg *config.Config) error {
 		klog.Fatalf("MicroShift must be run privileged")
 	}
 
+	startLog := startuplogger.NewStartupLogger()
+
 	klog.InfoS("MICROSHIFT STARTING")
 	microshiftStart := time.Now()
 
+	startLog.LogMicroshiftStart(microshiftStart)
```

Comment:
I think that we could move the log and start timestamp to the start logger, so the `startLog.LogMicroshiftStart()` would emit the log and save the time.

Reply:
good idea, moved